cfy_manager/components/nginx/config/rest-location.cloudify

```diff
@@ -6,6 +6,11 @@ location ~ ^/api/v(1|2|2\.1|3|3\.1)/(blueprints|executions|deployments|nodes|eve
     proxy_set_header   X-Real-IP        $remote_addr;
     proxy_set_header   X-Server-Port    $server_port;
     proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
+
+    gzip on;
+    gzip_types application/json;
+    gzip_min_length 1000;
```

Comment:
i think this should be much bigger, we don't want to waste cpu on every request.

Reply:
i suggest 20k.
1000 bytes is very small, even if you compress down 90%, you save less than 1k, and it's not worth the cpu.

Reply:
If most of the UI responses are less than 20kb, then this change won't help UI users.
The gzip compression "strength" that we use is `1` (which is the default. The range is 1-9, when 9 is the highest compression level). I think we're not wasting CPU at all, even for 1kb, and I think that 20kb is way too high.

Maybe what we can do is this:

* Set up a low threshold (say, 1000 or 2000 bytes) for requests coming from the UI
* Set up a higher threshold for requests coming from anywhere else

That way, UI users will enjoy the best experience possible (bandwidth is much more critical for UI), and REST users will get something in the middle (I still think that 20kb is way too high. Maybe 10kb?).